dev-python/nuitka/nuitka-0.6.0.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6 )
```

Comment:
No 3.7?

Reply:
[scons in gentoo][https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-util/scons/scons-3.0.1-r100.ebuild] does not support python3_7, that's why I excluded it.